stdlib/public/stubs/UnicodeNormalization.cpp

```diff
@@ -78,31 +78,6 @@ UBool u_isdefined(UChar32);
 #include <mutex>
 #include <assert.h>
 
-static const UCollator *MakeRootCollator() {
-  UErrorCode ErrorCode = U_ZERO_ERROR;
-  UCollator *root = ucol_open("", &ErrorCode);
-  if (U_FAILURE(ErrorCode)) {
-    swift::crash("ucol_open: Failure setting up default collation.");
-  }
-  ucol_setAttribute(root, UCOL_NORMALIZATION_MODE, UCOL_ON, &ErrorCode);
-  ucol_setAttribute(root, UCOL_STRENGTH, UCOL_TERTIARY, &ErrorCode);
-  ucol_setAttribute(root, UCOL_NUMERIC_COLLATION, UCOL_OFF, &ErrorCode);
-  ucol_setAttribute(root, UCOL_CASE_LEVEL, UCOL_OFF, &ErrorCode);
```

Comment:
I suspect that removing these may allow us to also remove some of the `#include`s above; perhaps `unicode/ucol.h` and/or `unicode/ucoleitr.h`?

Reply:
I haven't tried. I just silenced the warning.